datumaro/plugins/coco_format/converter.py

```diff
@@ -576,10 +640,12 @@ def apply(self):
                             '' if self._merge_images else subset_name))
                     else:
                         log.debug("Item '%s' has no image info", item.id)
+                segmentation_path = osp.join(self._ann_dir,
+                    'panoptic' + ('' if self._merge_images else "_"+ subset_name))
                 for task_conv in task_converters.values():
                     task_conv.save_image_info(item,
                         self._make_image_filename(item))
```

Comment:
Probably we also need to fix handling of image paths wrt `self._merge_images`.

Reply:
Don't get it